pl_bolts/datamodules/kitti_datamodule.py

```diff
@@ -56,6 +60,11 @@ def __init__(
             batch_size: the batch size
             seed: random seed to be used for train/val/test splits
         """
+        if not _TORCHVISION_AVAILABLE:
+            raise ModuleNotFoundError(  # pragma: no-cover
```

Comment:
Why is this happening here instead of at the top of the file?

Reply:
That's because we want to defer raising `ModuleNotFoundError`. If we raise `ModuleNotFoundError` at the top of the file, `from pl_bolts.datamodules import ...` fails even if the requirements for the submodule are met (which was reported in #291 and fixed in #303).

Reply:
@annikabrundyn it is because if importing, you are allowed to import the module, you just cannot execute it...